### PR TITLE
Fixes #14599: Rudder postgres unit test are broken

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/14599

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/test/resources/database.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 jdbc.driverClassName = org.postgresql.Driver
-jdbc.url = jdbc:postgresql://localhost:5432/test
-jdbc.username = test
-jdbc.password = test
+jdbc.url = jdbc:postgresql://localhost:55432/test
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-core/src/test/resources/database.properties]: Isn't that specific to your own environement ?

Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/test/resources/database.properties]: totally!

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/repository/jdbc/ReportsTest.scala [hunk 2]
```diff
@@ -144,26 +144,27 @@ class ReportsTest extends DBCommon {
     val reports = (
       Map[String, Seq[Reports]]() +
       node("n0")(
-          ("hasPolicyServer-root", "d1", 1, "common", "EndRun", run1, "result_success", "End execution")
+          // ruleId  dirId serial comp  keyVal  execTime   severity      msg
+          ("rudder", "run", 0, "end", "", run1, "control", "End execution")
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/repository/jdbc/ReportsTest.scala]: :+1:

**Comment by fanf** [comment]
Commit modified

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/repository/jdbc/ReportsTest.scala (hunk 2 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/repository/jdbc/ReportsTest.scala]: it's so much better :))

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**